frontend/src/app/components/modals/editor/macro-wp-button-modal/wp-button-macro.modal.ts

```diff
@@ -40,7 +40,8 @@ import {OpModalLocalsMap} from "core-components/op-modals/op-modal.types";
 import {I18nService} from "core-app/modules/common/i18n/i18n.service";
 import {TypeResource} from "core-app/modules/hal/resources/type-resource";
 import {CurrentProjectService} from "core-components/projects/current-project.service";
-import {WorkPackageDmService} from "core-app/modules/hal/dm-services/work-package-dm.service";
+import {APIV3Service} from "core-app/modules/apiv3/api-v3.service";
+import {FormResource} from "core-app/modules/hal/resources/form-resource";
 
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><del><span data-href="https://github.com/opf/openproject/blob/6f96d07af76f908fb7d38016c6f7731d39a28086/frontend/src/app/components/modals/editor/macro-wp-button-modal/wp-button-macro.modal.ts#L45"></span>Please use <code>ChangeDetectionStrategy.OnPush</code> for this component</del></td>
    </tr>
  </tbody>
</table>